pgbedrock/context.py

```diff
@@ -215,7 +215,8 @@
     SELECT
         substring(version from 'PostgreSQL ([0-9.]*) ') AS postgres_version,
         substring(version from 'Redshift ([0-9.]*)') AS redshift_version,
-        version LIKE '%Redshift%' AS is_redshift
+        version LIKE '%Redshift%' AS is_redshift,
+        exists(SELECT * FROM pg_roles WHERE rolname = 'cloudsqladmin') AS is_google_cloud
```

Comment:
Looks like this is the only way to detect cloudsql, `versions()` doesn't reveal much
![image](https://user-images.githubusercontent.com/221981/214254672-3960d7c0-720b-454f-a887-d0ff4d4195fa.png)